src/cls.js

```diff
@@ -53,6 +53,17 @@ module.exports = {
 
   setRootContext: function setRootContext(rootContext) {
     getNamespace().set('root', rootContext);
+  },
+
+  getTransaction: function getTransaction() {
```

Comment:
It would be good to unify the 'transaction' and 'root' as there is a 1-1 relationship between them.